fs/vfs/fs_fcntl.c

```diff
@@ -46,7 +46,7 @@
  * Name: file_vfcntl
  ****************************************************************************/
 
-static int file_vfcntl(FAR struct file *filep, int cmd, va_list ap)
+static int file_vfcntl(FAR struct file *filep, int cmd, va_list ap, int fd)
```

Comment:
remove fd, we should move the handle of F_DUPFD/F_DUPFD_CLOEXEC/F_GETFD/F_SETFD to fcntl and call dup instead